Source/Krypton Components/Krypton.Toolkit/Palette Builtin/Base/PaletteBase.cs

```diff
@@ -1231,7 +1231,15 @@ public virtual float BaseFontSize
         {
             get => _baseFont ?? new("Segoe UI", BaseFontSize, FontStyle.Regular);
 
-            set => _baseFont = value;
+            set
+            {
+                _baseFont = value;
+
+                if (value == null)
+                {
+                    _baseFont = new("Segoe UI", BaseFontSize, FontStyle.Regular);
```

Comment:
This means that the code can accept a null but not return it (See above) so mark with `[AllowNull]`

Reply:
Done